sonaranalyzer-dotnet/its/sources/ManuallyAddedNoncompliantIssues/NetFramework48/NetFramework48.csproj

```diff
@@ -48,7 +48,6 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Include="BinaryFormatterTest.cs" />
-    <Compile Include="InsecureDeserialization.cs" />
```

Comment:
why do you want to remove it? you can leave it and replace the `Noncompliant` comment with `// we don't raise issues when not running with the S4MSB`

and maybe open a ticket to add a manual safety net for hotspots in the Java IT, where we integrate with SonarQube